samples/matter/light-bulb/readme.rst

```diff
@@ -0,0 +1,150 @@
+.. _matter-light-bulb:
+
+Matter Light bulb sample
+########################
+
+Overview
+********
+
+Sample for how to control a green dimmable light bulb endpoint and Matter SDK.
+Sample support BLE commisioning to Thread network.
+
+Requirements
+************
+
+- Alif Balletto Development Kit
+- Matter Thread Border router and Matter Controller
+- Pre provisioned Matter Light sample or 3rd party light switch
+
+Build Sample
+************
+
+Enable automatic start of the openthread stack
+
+.. code-block:: console
+
+   west build -p always -b alif_b1_dk_rtss_he samples/matter/light-bulb
+
+Light-bulb endpoint
+*********************
+
+Sample have generated ZAP and Matter file which enable with Matter Root endpoint 0 and OnOff Dimmable Light bulb endpoint 1.
+
+User interface
+**************
+
+Devkit use 2 leds for indicating Thread and BLE communication state
+
+PWM led (Green Led):
+- Dimmable Light controlled by Matter controller or binded On/Off light switch
+
+Blue LED:
+
+- Blinking slowly indicate that Matter BLE Advertisment is active for Matter Provisioning
+- Fast Blink (100ms) Means that BLE connection is active and Matter Provisioning is ongoing
+
+Red LED:
+
+- Fast Blinking (100ms) indicate Thread network is active for joining provisionmed newtwork
+- 1 Second constant Active Mode indicate That Matter controller subscribed is establis,ent or resumed
+- Single short active indicate Endpoint activation
+
+Sample support Balletto Devkit Joystick buttons
+
+Center Button `SW0`:
+
+- Hold over 3 seconds for doing Matter factoryreset. After Factory reset press DK normal Reset Button for activate BLE advertisment.
+
+Provisioning with Apple Home
+******************************************
+
+1. Flash builded binary to device and press ``Reset`` -button.
+#. After reset you shuold see slowly blinking blue led which indicate that Device is advertisment Matter ready device for provision.
+#. Open ``Apple Home`` application with Iphone or Ipad which is connected to same Wifi with Matter Conttroler and Thread border router.
+#. Click ``+`` -button from app for `Add Accessory`.
+#. Scan QR code:
+
+   .. image:: QRCode.png
+
+#. Click ``Add To Home`` for Start provisioning to Thread neywork and Matter fabric.
+#. You shuold see now fast blinking Blue led when BLE connection is established.
+#. ``Aplle Home`` will warning Uncertificated Acessory so just click ``Add Anyway`` for continue.
+#.  When Provision is ready ``Apple Home`` ask where to add light select a room and click ``Continue``.
+#. Now you can rename a light default is ``Matter Accessory`` name for example ``Matter Light`` and click ``Continue``.
```

Comment:
Now you can rename a light default is ->
You can rename the default name to ``Matter Light`` and click ``Continue``.